bokeh/charts/chart.py

```diff
@@ -82,14 +82,14 @@ class Chart(Plot):
     def __init__(self):
 
         # Initializes then gets default properties
-        super(Chart, self).__init__(id=self._options.id or make_id())
+        # NOTE: we use option._id here since ChartOptions converts id arg to _id
+        super(Chart, self).__init__(id=self._options._id or make_id())
```

Comment:
This has sort of historical reasons... You've already got the related part:

https://github.com/bokeh/bokeh/blob/master/bokeh/charts/chart.py#L54-L73

Honestly I don't remember why `ChartOptions` is a Model... maybe @bryevdv remembers? We kind of worked together on this at the time...

You are right though, I shouldn't use the same id.. your suggestion of using `ChartOptions.properties()` is a better one. Thanks